engagement_database/data_models.py

```diff
@@ -0,0 +1,106 @@
+import uuid
+
+from core_data_modules.data_models import Label
+
+
+class MessageStatuses(object):
+    LIVE = "live"              # Message is part of the 'live' data, and should be used in analysis.
+    STALE = "stale"            # Message is stale and we should attempt to get a newer answer if we can. This message
+                               # can be used until we get a newer answer.
+    ARCHIVED = "archived"      # This message is no longer relevant and should be ignored by analysis.
+    TOMBSTONED = "tombstoned"  # Message is tombstoned, so personally identifiable details have been removed.
```

Comment:
I suggest using `NON_PII = "Non-PII"` for Non-personally identifiable information.

Reply:
How about "anonymised"?